test/functional/feature_dbcrash.py

```diff
@@ -190,34 +186,36 @@ def generate_small_transactions(self, node, count, utxo_list):
         num_transactions = 0
         random.shuffle(utxo_list)
         while len(utxo_list) >= 2 and num_transactions < count:
-            tx = CTransaction()
-            input_amount = 0
-            for _ in range(2):
-                utxo = utxo_list.pop()
-                tx.vin.append(CTxIn(COutPoint(int(utxo['txid'], 16), utxo['vout'])))
-                input_amount += int(utxo['amount'] * COIN)
-            output_amount = (input_amount - FEE) // 3
-
-            if output_amount <= 0:
+            utxos_to_spend = [utxo_list.pop() for _ in range(2)]
+            input_amount = int(sum([utxo['value'] for utxo in utxos_to_spend]) * COIN)
+            if input_amount < FEE:
                 # Sanity check -- if we chose inputs that are too small, skip
                 continue
 
-            for _ in range(3):
-                tx.vout.append(CTxOut(output_amount, bytes.fromhex(utxo['scriptPubKey'])))
+            tx = self.wallet.create_self_transfer_multi(
+                from_node=node,
+                utxos_to_spend=utxos_to_spend,
+                num_outputs=3,
+                fee_per_output=FEE // 3)
 
-            # Sign and send the transaction to get into the mempool
-            tx_signed_hex = node.signrawtransactionwithwallet(tx.serialize().hex())['hex']
-            node.sendrawtransaction(tx_signed_hex)
+            # Send the transaction to get into the mempool
+            node.sendrawtransaction(tx.serialize().hex(), 0)
```

Comment:
What about setting this to `0` by default for MiniWallet?

Reply:
Also, it seems confusing that this needs to be set at all. Why would a fee of `1000` sat be ever too high?